docs/virtual_machines/accessing_virtual_machines.md

```diff
@@ -290,21 +216,170 @@ spec:
           propagationMethod:
             qemuGuestAgent:
               users:
-              - "fedora"
+              - fedora
       volumes:
-      - dataVolume:
-          name: fedora-dv
-        name: disk0
+      - containerDisk:
+          image: quay.io/containerdisks/fedora:latest
+        name: containerdisk
       - cloudInitConfigDrive:
-          userData: |
-            #!/bin/bash
+          userData: |-
+            #cloud-config
+            password: fedora
+            chpasswd: { expire: False }
+            # Disable SELinux for now, so qemu-guest-agent can write the authorized_keys file
+            bootcmd:
+              - setenforce 0
+        name: cloudinitdisk
+EOF
+```
+
+### Accessing SSH with virtctl
+
+Every `VirtualMachineInstance` provides a `virtualmachineinstances/portforward`
+subresource, that can be used to create a websocket backed network tunnel to
+a port inside the instance similar to Kubernetes pods.
+
+One use-case for this subresource is to forward SSH traffic into
+the `VirtualMachineInstance` either from the CLI or a web-UI.
 
-            sudo setenforce Permissive
-            sudo yum install -y qemu-guest-agent
-            sudo systemctl start qemu-guest-agent
-        name: disk1
-END
+To connect to a `VirtualMachineInstance` from your local machine, `virtctl`
+provides a lightweight SSH client with the `ssh` command, that uses port
+forwarding. Refer to the command's help for more details.
 
-kubectl create -f my-vm.yaml
+```shell
+virtctl ssh
```

Comment:
```suggestion
virtctl ssh <virtual-machine-instance-name>
```
Or something similar that makes clear that you have to provide the VMI name.

Reply:
I deliberately left it out so the mentioned help would get displayed.